### PR TITLE
Add Web infrastructure

### PR DESCRIPTION
**Pull request by vetletm**
Took code written for Consul infrastructure, changed to fit web infrastructure with appropriate naming and IP addressing.

### CODE REVIEW

**Comment by vetletm** [comment]
Fixes #10

File: terraform/modules/web/main.tf [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# create a new web instance
+resource "openstack_compute_instance_v2" "web_server" {
+  count       = "${var.replicas}"
+  name        = "web${count.index+1}"
+  image_name  = "${var.image_name}"
+  flavor_name = "${var.flavor_name}"
```
Comment (p3lim) [inline terraform/modules/web/main.tf:6]: Missing formatting/linting.

**Review by p3lim — DISMISSED** [review]
LGTM, run `terraform fmt` first.

File: terraform/modules/web/main.tf (hunk 1 above)
Comment (vetletm) [reply terraform/modules/web/main.tf:6]: LGTM.

**Pull request merged**